showcase/src/assets/translations/nl.json

```diff
@@ -18,6 +18,18 @@
         "PARENT": "Ouder",
         "CHILD": "Kind"
       },
+      "CARD": {
+        "ADVANCED": {
+          "DETAILS": "Stark handles more complex cards also, here is an example with subtitle, actions and also the option to have full width elements thanks to the <code>.stark-full-width</code> class added to the <code>mat-card-content</code> tag.",
+          "TITLE": "Advanced card"
+        },
+        "BASIC": {
+          "DETAILS": "Stark provides an alternative layout for cards. You can keep the same html structure and add the <code>.stark-card</code> class to the Angular Material card. We also recommend to use <code>h3</code> tag inside the <code>mat-card-title</code>.<br>For more information on Angular Material's card, check out this link: <a href=\"https://material.angular.io/components/card/overview\" target=\"_blank\">https://material.angular.io/components/card/overview</a>",
+          "TITLE": "Basic card"
+        },
+        "CODE": "Here, the code to display this card:",
+        "TITLE": "Stark Cards"
+      },
```

Comment:
"CARD": {
         "ADVANCED": {
           "DETAILS": "Stark kan ook complexere kaarten aan. Hier is een voorbeeld met een ondertitel en acties. Er is ook de mogelijkheid om elementen de volledige breedte te laten gebruiken dankzij de <code>.stark-full-width</code> klasse die toegevoegd is aan de <code>mat-card-content</code> html-tag.",
           "TITLE": "Geavanceerde kaart"
         },
         "BASIC": {
           "DETAILS": "Stark biedt een alternatieve opmaak aan voor kaarten. Je kan de html structuur behouden en de klas <code>.stark-card</code> aan de Angular Material kaart toevoegen. We raden aan om het <code>h3</code> label te gebruiken binnen een <code>mat-card-title</code>.<br>Meer informatie over de kaarten van Angular Material, kan je terugvinden via deze link: <a href=\"https://material.angular.io/components/card/overview\" target=\"_blank\">https://material.angular.io/components/card/overview</a>",
           "TITLE": "Basiskaart"
         },
         "CODE": "Hier is de code om een kaart te tonen:",
         "TITLE": "Stark Kaarten"

Reply:
Thanks!